bpf/lib/common.h

```diff
@@ -235,7 +235,9 @@ struct policy_key {
 
 struct policy_entry {
 	__be16		proxy_port;
-	__u16		pad0;
+	__u8		deny:1,
```

Comment:
Not quite, this change needs to be part of this commit or otherwise the CLI command to add and remove bpf entries wouldn't have any effect as well as cilium wouldn't run due mismatch of the bpf field "pad0" and the "deny" field in the controlplane.